inspectors/view/qml/notation/timesignatures/TimeSignaturePopup.qml

```diff
@@ -0,0 +1,101 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import MuseScore.Inspectors 3.3
+import QtQuick.Layouts 1.3
+import "../../common"
+
+StyledPopup {
+    id: root
+
+    property QtObject model: null
+
+    implicitHeight: contentColumn.implicitHeight + topPadding + bottomPadding
+    width: parent.width
+
```

Comment:
There should be an icon property around here. Is the icon not ready yet?

Reply:
I don't see one in [KeySignaturePopup.qml](https://github.com/vpereverzev/MuseScore/blob/0dd6aa0ba73f93802548802c6cdbe5d2efea171c/inspectors/view/qml/notation/keysignatures/KeySignaturePopup.qml).

Reply:
I've simplified the structure of proxy models and the new popups declaration at the beginning of the week